src/amo/components/Page/index.js

```diff
@@ -60,6 +63,13 @@ export const PageBase = ({
           (!isHomePage ||
             !enableFeatureHeroRecommendation ||
             clientApp === CLIENT_APP_ANDROID) && <AppBanner />}
+          {// Exclude the WrongPlatformWarning from the home page if it will be
+          // included via HeroRecommendation or if showWrongPlatformWarning is
+          // false, but include it on the Android home page.
+          (!isHomePage ||
+            !enableFeatureHeroRecommendation ||
+            clientApp === CLIENT_APP_ANDROID) &&
+            showWrongPlatformWarning && <WrongPlatformWarning />}
```

Comment:
Did you consider implementing this with [`AddonCompatibilityError`](https://github.com/mozilla/addons-frontend/blob/master/src/amo/components/AddonCompatibilityError/index.js) instead? I know it's a bit different, just curious.